Dockerfile.dl-glove-6B-50d

```diff
@@ -0,0 +1,38 @@
+FROM elifesciences/sciencebeam-trainer-delft-grobid_unstable:04346b26b5fa7966cef4d6ec6bba867feba52d66-cbb7e63d1b2c27c67afe591abbb23ccc95c6cfb7
```

Comment:
i'd prefer to give use a stable image with some descriptive tag (or version number) as the base image, and if that is not possible, probably use an unstable image with some descriptive tag to show that is the point of divergence

Reply:
same comment goes for `Dockerfile.dl-no-word-embeddings`

Reply:
It certainly would look better using a release. Currently there is no release. It may be a can of worms as if I create a release of `sciencebeam-trainer-delft-grobid_unstable` then the base image it uses should be a release too. Which would be `elifesciences/grobid`. We would have to decide how to name the versions of that without confusing release versions of mainline grobid. Maybe `0.5.6.1` where `0.5.6` is the upstream grobid release.

Could we agree to do that outside this PR and create a separate issue for it?